numpy/random/_generator.pyx

```diff
@@ -238,6 +238,27 @@ cdef class Generator:
         """
         return self._bit_generator
 
+    def spawn(self, int n_children):
+        """
+        Create new independent child generators.
+
+        This is a convenience method to safely spawn new random number
+        generators via the `numpy.random.SeedSequence.spawn` mechanism.
+        The original seed sequence is used by the bit generator and accessible
+        as ``Generator.bit_generator.seed_seq``.
```

Comment:
I'm not sure what this sentence is trying to convey in this context. I'm not sure why the address of the `SeedSequence` is important here. On first reading, it read like it was claiming that the original `SeedSequence` was available on the _spawned_ generators. I think this might work better as an `Example` rather than prose, if it needs to be said in this docstring.

Reply:
Felt like pointing where things come from.  But, I agree that it is probably easy enough to find when a user actually wants to (and the vast majority shouldn't).

Instead added a `See Also` section, that seems more useful.

I gave a shot expanding the docs a bit.  The main info about spawning is currently in the bit generator, right now only inserted the new approach there.

(Noted `jumped` in `BitGenerator.spawn()`, not sure it is helpful, and maybe `See Also` of `jumped` should point back to spawn?)

Reply:
The main part of the narrative docs I was thinking of was the [Parallel RNG](https://numpy.org/devdocs/reference/random/parallel.html#seedsequence-spawning) page. It could wait for another doc-focused PR, though.

Reply:
🤦 sorry, gave it a shot to expand those a bit minimally, since that is really the right place to link to.  Not sure that is the best approach so happy to try and redo more.